front/app/containers/ProjectsShowPage/shared/pb/PBExpenses.tsx

```diff
@@ -259,7 +259,7 @@ const TooltipContentText = styled.div`
 `;
 
 interface Props {
-  participationContextId: string | null;
+  participationContextId: string;
```

Comment:
It's a string everywhere we pass `participationContextId` so better to narrow the type.